cnxeasybake/tests/rulesets/deferred.less

```diff
@@ -0,0 +1,21 @@
+// Test before pseudo target
+
+div[data-type='chapter'] {
+    section.practice-test {
+        string-set: childclasses string(childclasses) " " attr(class);
+    }
+    section.key-equations {
+        string-set: childclasses string(childclasses) " " attr(class);
+    }
+
+    &:deferred {
```

Comment:
Is this meant to work for something like the ToC (instead of `move-to:` and `content: pending-deferred()`)?